### PR TITLE
Update test.en-any.en

### PR DESCRIPTION
**Pull request by Jamiil92**
added some random subset of my dataset en-ddn

### CODE REVIEW

**Comment by jaderabbit** [comment]
So, I'm contemplating the practicality of doing this. 

By updating the english test set, it is now invalid to compare the existing BLEU results for all benchmarks to new models that might come along. 

i.e. we'd need to refilter test sets and retrain EVERY benchmark to make sure that we can compare. 

Is there a particular reason you chose to augment the test set?

**Comment by Jamiil92** [comment]
No specific reason.I was just following the instructions of the cell below.

`# Download the global test set.
! wget https://raw.githubusercontent.com/masakhane-io/masakhane/master/jw300_utils/test/test.en-any.en
  
# And the specific test set for this language pair.
os.environ["trg"] = target_language 
os.environ["src"] = target_language 

! wget https://raw.githubusercontent.com/masakhane-io/masakhane/master/jw300_utils/test/test.en-$trg.en 
! mv test.en-$trg.en test.en
! wget https://raw.githubusercontent.com/masakhane-io/masakhane/master/jw300_utils/test/test.en-$trg.$trg 
! mv test.en-$trg.$trg test.$trg

# TODO: if this fails it means that there is NO test set for your language yet. It's on you to create one.
# A good idea would be to take a random subset of your data, and add it to https://raw.githubusercontent.com/masakhane-io/masakhane/master/jw300_utils/test/test.en-any.en.
# Make a Pull Request and get it approved and merged.
# Then repeat this cell to retrieve the new test set.
# Then proceed to the next cell that will filter out all duplicates from the training set, so that there is no overlap between training and test set.`

Hower i got around this. 
and finally trained the en-ddn baseline model.

**Comment by Jamiil92** [comment]
i have just removed my random subset from the test set.

**Comment by Jamiil92** [comment]
Is it possible to merge the other commit from this PR ?
Or i should simply open a new PR?
I've got a baseline for English to Dendi.

**Comment by juliakreutzer** [comment]
Is there any overlap of your data and the global test data? If so, take this as test set. 
If not, we need to figure out how to do this, like @jaderabbit said it might be problematic at this point in time. 
The description in the notebook was maybe a bit naive, I'm sorry about that. For now I'd suggest you add your custom test set in addition to the global one without merging it, and we know that we shouldn't use this model for transfer learning without care.

**Comment by Jamiil92** [comment]
Just added the custom test set

**Comment by juliakreutzer** [comment]
Great, thanks so much! Would you mind adding your results to language_pairs.md as well? Thanks!

**Comment by Jamiil92** [comment]
language_pairs.md updated

**Comment by Jamiil92** [comment]
hello..  @juliakreutzer and @jaderabbit ..

**Comment by juliakreutzer** [comment]
Thanks for the patience, @Jamiil92!
